guides/common/modules/proc_creating-a-rolling-content-view.adoc

```diff
@@ -0,0 +1,44 @@
+[id="creating-a-rolling-content-view"]
+= Creating a rolling content view
+
+Use this procedure to create a rolling content view.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-rolling-content-view[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Lifecycle* > *Content Views*.
+. Click *Create content view*.
+. In the *Create content view* window, enter a name for the content view in the *Name* field.
+{Project} automatically completes the *Label* field from the name you enter.
+. Optional: In the *Description* field, enter a description of the content view.
+. On the *Type* tab, select *Rolling content view*.
+. Click *Create content view*.
+. Click *Show repositories*.
+. Select the repositories that you want to add to your rolling content view.
+. Click *Add repositories* to add all selected repositories to your rolling content view.
+
+[id="cli-creating-a-rolling-content-view"]
+.CLI procedure
+. Before you create a rolling content views, list all available repositories:
```

Comment:
```suggestion
. List all available repositories to identify IDs of repositories to add to your rolling content view:
```
The order of steps is clear. No need for the "Before..." bit. Instead, let's tell them why they are doing it.

Reply:
Applied.